_guides/page-objects.md

```diff
@@ -202,11 +202,10 @@ See also:
 **Blog postings related to Page Objects and Watir-webdriver**
 * cheezyworld's series on UI testing [part 1](http://www.cheezyworld.com/2010/11/09/ui-tests-not-brittle/), [part 2](http://www.cheezyworld.com/2010/11/13/ui-tests-part-two/), [part 3](http://www.cheezyworld.com/2010/11/19/ui-tests-introducing-a-simple-dsl/), [part 4](http://www.cheezyworld.com/2010/11/21/ui-tests-default-dat/), [part 5](http://www.cheezyworld.com/2010/12/16/ui-tests-putting-it-all-together/)
 * Watermelon blog article on ['roll your own' page objects](http://watirmelon.com/2012/06/04/roll-your-own-page-objects/)
+* [WatirPump: a new Page Object library for Ruby and Watir](https://blog.bwilczek.eu/introducing-watirpump-a-new-page-object-library-for-ruby-and-watir)
```

Comment:
I would not use a word "new" in here since it will be not new after some time ;)

Reply:
I've fixed the wording, I agree that using "new" can be misleading :)